### DISCUSSION TITLE
need help ( how to start or test the application locally)

### DISCUSSION THREAD

**Discussion by rahulptl165** [camicroscope/caMicroscope#1039] [General]
Hi team!

I’ve set up the "caMicroscope" repository on my computer and followed the steps in CONTRIBUTING.md by running npm install and npm run lint. However, I didn’t see any instructions on how to actually start or test the application locally. Could you let me know if there’s a way to run the application locally ?

**Comment by birm** [comment]
caMicroscope is currently deployed as a set of docker containers; see https://github.com/camicroscope/distro 

Please let me know if anything in the docs (http://camicroscope.org/docs/#setup-installation-and-hosting) needs clarification!
